nebula.config.js

```diff
@@ -9,4 +9,181 @@ module.exports = {
       "process.env.PACKAGE_VERSION": JSON.stringify(version),
     },
   },
+  serve: {
+    flags: {},
+    keyboardNavigation: false,
+    themes: [
```

Comment:
Added 3 different themes.
- The "default" has all default values
- "Demo" has a unique value for each property
- "Only transparent borders" show case when cell borders are turned off